DigitalLearningSolutions.Web/ViewComponents/FilterableTagsViewComponent.cs

```diff
@@ -0,0 +1,14 @@
+﻿namespace DigitalLearningSolutions.Web.ViewComponents
+{
+    using System.Collections.Generic;
+    using DigitalLearningSolutions.Web.ViewModels.Common;
+    using Microsoft.AspNetCore.Mvc;
+
+    public class FilterableTagsViewComponent : ViewComponent

```

Comment:
Could this be a partial instead if all we need is rendering a view with the model?